recipes/py-hunspell/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "py-hunspell" %}
```

```suggestion
{% set name = "pyhunspell" %}
```
Is it ok if we name match the GitHub repo since that is where the source is developed?

---

Of course, I just used that named following the name convention of other packages I saw before. I agree this name is better because of the GitHub repo name